.github/workflows/release.yml

```diff
@@ -0,0 +1,28 @@
+name: Add Tags to Release
+
+on:
+  release:
+    types: [published]
+
+jobs:
+  release:
+    runs-on: ubuntu-latest
+    steps:
```

Comment:
Please include the build step to build the image if missing - similar to what we do for the backend (and now frontend). This to support emergency releases/tags that may not go through the `main` branch.

Reply:
added build-if-needed to match backend

Reply:
👍🏻